main/webapp/modules/core/scripts/index/default-importing-sources/sources.js

```diff
@@ -100,7 +100,14 @@ UrlImportingSourceUI.prototype.attachUI = function(bodyDiv) {
     }
   });
   this._elmts.addButton.click(function(evt) {
-    self._elmts.buttons.before(self._elmts.urlRow.clone());
+    let newRow = self._elmts.urlRow.clone();
+    let trashButton = $('<a href="javascript:{}" title="Remove this row" bind="removeUrlButton"><span class="ui-icon ui-icon-trash"></span></a>');
```

Comment:
The `title` could ideally be localized and the `bind=` is maybe not necessary since you are directly binding the event handler below.